src/rlx_release.erl

```diff
@@ -226,30 +232,32 @@ format(Release) ->
     format(0, Release).
 
 -spec format(non_neg_integer(), t()) -> iolist().
-format(Indent, #release_t{name=Name, vsn=Vsn, erts=ErtsVsn, realized=Realized,
-                         goals = Goals, applications=Apps}) ->
+format(Indent, #release_t{name=Name,
+                          vsn=Vsn,
+                          erts=ErtsVsn,
+                          realized=Realized,
+                          goals = Goals,
+                          applications=Apps}) ->
     BaseIndent = rlx_util:indent(Indent),
     [BaseIndent, "release: ", rlx_util:to_string(Name), "-", Vsn, "\n",
-     rlx_util:indent(Indent + 2), " erts-", ErtsVsn,
-     ", realized = ",  erlang:atom_to_list(Realized), "\n",
+     rlx_util:indent(Indent + 1), "erts: ", ErtsVsn, "\n",
      rlx_util:indent(Indent + 1), "goals: \n",
-     [[rlx_util:indent(Indent + 2),  format_goal(Goal), ",\n"] || Goal <- Goals],
+     [[rlx_util:indent(Indent + 2),  format_goal(Goal), "\n"] || {_, Goal} <- maps:to_list(Goals)],
```

Comment:
can use `maps:values/1` here and skip the tuple match bit

Reply:
Derp, yup, thanks, fixing.